LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Looks good for now.

- [ ] Should set the license in the pyproject.toml as well
- [ ] We should verify this is the correct license / no possible issues and change/squash before we make the repo public.